### PR TITLE
feat: enable different ways to show matrix in LaTeX

### PR DESCRIPTION
**Pull request by joao-guilherme**
LaTeX has many ways to display an matrix, let's use.

image

### CODE REVIEW

File: nerdamer.core.js [hunk 1]
```diff
@@ -10042,14 +10042,15 @@ var nerdamer = (function (imports) {
         text: function () {
             return 'matrix(' + this.elements.toString('') + ')';
         },
-        latex: function (option) {
+        latex: function (type = 'v', options) {
```
Comment (jiggzson) [inline nerdamer.core.js]: Please make type the second argument. Making it the first argument will break existing code. Also can you use `type = type || 'v'` or something similar for default options so the code doesn't break in older browsers?

**Comment by jiggzson** [comment]
@joao-guilherme, Thanks. That's a great idea. I've added a comment to the PR so as soon as you make the requested changes I can close this.

**Comment by jiggzson** [comment]
@joao-guilherme, I was thinking that maybe we can reuse the options parameter. Just so we don't break existing code, we can check if it's a string or an object. If it's a string, proceed as usual. If it's an object we can look for the `matrix_type` key.  What do you think?

**Comment by joao-guilherme** [comment]
@jiggzson, I'm new here, but looking at the code, I prefer your first idea, because making the _option_ an object adds more complexity than moving the _type_ to the second parameter.

I didn't see where we can break the code, since passing a single parameter, _option_ will have the correct value and _type_ will be undefined

**Comment by jiggzson** [comment]
Closing. No response given and PR breaks test cases. Added for consideration in #358.

**Pull request closed without merging**